README.md

```diff
@@ -206,9 +206,9 @@ sqlglot.transpile("SELECT foo( FROM bar")
 ```
 
 ```
-sqlglot.errors.ParseError: Expecting ). Line 1, Col: 13.
+sqlglot.errors.ParseError: Expecting ). Line 1, Col: 20.
   select foo( FROM bar
-              ~~~~
+                   ~~~
```

Comment:
The reason this column changed is because the parser became more lenient: you can now do `SELECT foo (FROM bar)` and that parses successfully because `FROM table` is valid DuckDB syntax.

This example may be a bit confusing though, it's not trivial to understand why SQLGlot underlines `bar` instead of `TABLE`. What do you say about changing this and the next one ("structured error") to this example instead?

```
>>> sqlglot.parse_one("SELECT foo FROM (SELECT x FROM t")
...
sqlglot.errors.ParseError: Expecting ). Line 1, Col: 32.
  SELECT foo FROM (SELECT x FROM t
                                 ~
```

Reply:
Hmm, I thought `FROM <table>` was already valid at that point, otherwise I'd expect the initial example to have a different parse error such as `Expected TABLE / SELECT` right after the opening parenthesis. 

The example you're proposing does seem easier to digest as it's more ansi-like /  not tied to DuckDB per se, I'll use it in my next commit

Reply:
Since `FROM` is a reserved keyword, I believe most other dialects would point to `FROM` as the culprit, e.g.:

```
sqlite> SELECT foo (FROM bar;
Parse error: near "FROM": syntax error
  SELECT foo (FROM bar;
              ^--- error here
```